### DISCUSSION TITLE
osgdb_zip.dll error

### DISCUSSION THREAD

**Discussion by kahlenberg** [pelicanmapping/osgearth#2547] [Support Q&A]
Hi,
I built osgearth on Windows 11, 64-bit with cmake 3.27.6, when I run `osgearth_version.exe --caps`, then I receive following error:
![image](https://github.com/gwaldron/osgearth/assets/6330648/2e6a3dbd-5b7c-4653-bcf6-8d52cc08313a)
Additionally, I get errors related to font files:

![image](https://github.com/gwaldron/osgearth/assets/6330648/34752d3c-a9f9-4743-afa5-3294f7ecb566)
I have the CascadiaCode font installed but couldn't find any solution.

Thanks for your help.

**Comment by gwaldron** [comment]
Most likely problem is that `zip.dll` is missing from your path.

The FontConfig error comes from OpenSceneGraph - you can safely ignore it.

**Reply by kahlenberg** [reply]
But it is not related to zip.dll, instead it is related to osgdb_zip.dll which is already in PATH.

**Reply by kahlenberg** [reply]
I checked the osgdb_zip.dll with Dependencies.exe (https://github.com/lucasg/Dependencies), the dependencies and paths are correct.
```
├ osgdb_zip.dll (ROOT) : lib\RelWithDebInfo\osgdb_zip.dll 
.....
 | ├  zip.dll (ApplicationDirectory) : lib\RelWithDebInfo\zip.dll 
...
```

**Reply by gwaldron** [reply]
Thanks. Not sure then. The only possibilities that come to mind are (a) a path problem, or (b) mismatched .lib and .dll files, or (c) mismatched release/debug builds.
